manifests/config.pp

```diff
@@ -2,7 +2,9 @@
 #
 # This class is called from opendaylight for service config.
 #
-class opendaylight::config {
+class opendaylight::config (
+   $odl_rest_port = $::opendaylight::params::odl_rest_port
```

Comment:
This is throwing a style check error (should be two space soft tabs): https://travis-ci.org/dfarrell07/puppet-opendaylight/jobs/49525399